tests/lean/608.hlean.expected.out

```diff
@@ -1,11 +1,11 @@
 definition id [reducible] : Π {A : Type}, A → A :=
-λ (A : Type) (a : A), a
+λ (A : Type) (H : A), H
```

Comment:
I have changed the export procedure. Now, it takes the binder names into account.